tools/integrations/manage.sh

```diff
@@ -231,11 +231,18 @@ cmd_publish() {
     # Alternative local approach @ https://github.com/docker/buildx/issues/301#issuecomment-755164475
     # We need to use the regular docker buildx driver (not docker container) because we need this intermediate contaiers to be available for later build steps
 
+
+    echo Installing arm64 docker emulation
+    docker run --privileged --rm tonistiigi/binfmt --install arm64
+
     for arch in $(echo $build_arch | sed "s/,/ /g")
     do
       echo "building base images for $arch"
```

Comment:
nit: this `echo` should be inside the if block, not outside: the script output shouldn't say that we're going to build base images unless we're actually going to build base images